src/fundus/publishers/base_objects.py

```diff
@@ -44,6 +46,7 @@ def __init__(
         self.url_filter = url_filter
         self.request_header = request_header
         self.deprecated = deprecated
+        self.robots = RobotFileParser(self.domain + "robots.txt")
```

Comment:
```suggestion
        self.robots = RobotFileParser(self.domain + "robots.txt" if self.domain.endswith("/") else self.domain + "/robots.txt")
```
We don't require domains to end with / potentially causing a malformed URL.